.docker/docker_run_git.sh

```diff
@@ -30,6 +30,11 @@ fi
 # From now, stop at error
 set -e
 
+if [ $PS_DEV_MODE -ne 1 ]; then
+  echo "\n* Disabling DEV mode ...";
+  sed -ie "s/define('_PS_MODE_DEV_', true);/define('_PS_MODE_DEV_',\ false);/g" /var/www/html/config/defines.inc.php
```

Comment:
To fully support `PS_DEV_MODE` environment variable, shouldn't you manage cases 
- `$PS_DEV_MODE -eq 1` => `define('_PS_MODE_DEV_', true)`
- `$PS_DEV_MODE -eq 0` => `define('_PS_MODE_DEV_', false)`
- anything else => invalid value

?

Reply:
`PS_DEV_MODE` has a default value and it's 1, so it's never empty. The only case not handled is when it's equal to 0 to disable it.
But maybe for next versions, we should change who to handle this update, this way of doing it is a workaround.

Reply:
My point was more about "never trust user inputs" but as you want.

Reply:
My comment was more about "never trust user inputs" ^^